portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,7 +37,8 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-  private int maxNumComments;
+  private final int defaultMaxNumComments = 7;
+  private int maxNumComments = defaultMaxNumComments;
```

Comment:
i think you'll want to leave this variable declaration in the doGet method, otherwise you'll risk a race condition where user 1 gets user 2's max number of comments.

Reply:
Ok, I'll move that.